beanstalk.tf

```diff
@@ -54,4 +60,16 @@ resource "aws_elastic_beanstalk_environment" "this" {
     name      = "VPCId"
     value     = local.vpc_id
   }
+
+  setting {

```

Comment:
We prefer using capabilities for autoscaling instead of baking into the app modules directly.
If an app doesn't want autoscaling, it's off by default
Each use case for autoscaling usually differs (request count vs cpu vs memory vs others)
It's easier for users to extend the autoscaling support with custom functionality

Reply:
Here is a PR in this app module that will enable the use of capabilities to configure the ASG: https://github.com/nullstone-modules/aws-beanstalk-app/pull/4